### PR TITLE
Fix update custom fields endpoint

### PR DESCRIPTION
**Pull request by wuz**
- Update fixtures to use correct base URL
- Add github action and version updates
- Fix tests
- Fix update custom fields endpoint

### CODE REVIEW

File: spec/greenhouse_io/api/client_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe GreenhouseIo::Client do
 
-  FAKE_API_TOKEN = '123FakeToken'
+  FAKE_API_TOKEN = 'cc585d5281b4a6f2e657c7e0df8e3d9d-3'
```
Comment (tyh835) [inline spec/greenhouse_io/api/client_spec.rb]: Left over credentials?

Comment (wuz) [reply spec/greenhouse_io/api/client_spec.rb]: Shit, yep thanks

Comment (wuz) [reply spec/greenhouse_io/api/client_spec.rb]: Fixed!

**Review by tyh835 — DISMISSED** [review]
Looks good! (crazy how it's `custom_field` how that one endpoint 😅 )

**Pull request merged**